manifests/o/osara/osara/2022.1pre-974,eb153591/osara.osara.yaml

```
@@ -1,9 +1,8 @@
-# Created using wingetcreate 1.0.4.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
-
-PackageIdentifier: osara.osara
-PackageVersion: 2022.1pre-951,52c55fb0
-DefaultLocale: en-US
-ManifestType: version
-ManifestVersion: 1.1.0
-
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: osara.osara
+PackageVersion: 2022.1pre-974,eb153591
```

```suggestion
PackageVersion: 2022.1.974
```
This would be a great place for some cleanup using the AppsAndFeaturesEntries